test/controllers/changeset_controller_test.rb

```diff
@@ -1578,11 +1578,11 @@ def test_query
     assert_response :success, "can't get changesets by closed-ness"
     assert_changesets [3, 5, 6, 7, 8, 9]
 
-    get :query, :closed => "true", :user => users(:normal_user).id
+    get :query, :closed => "true", :user => users(:normal_user)
```

Comment:
This one is a query parameter and probably really should be ID and in fact I'm mystified as to how this is working because the controller code definitely wants a number for that parameter so the test framework must be converting it back to an ID as best I can tell.